be/src/exec/vectorized/topn_node.cpp

```diff
@@ -132,9 +133,16 @@ Status TopNNode::_consume_chunks(RuntimeState* state, ExecNode* child) {
 
     ScopedTimer<MonotonicStopWatch> timer(_sort_timer);
     if (_limit > 0) {
-        _chunks_sorter =
-                std::make_unique<ChunksSorterTopn>(&(_sort_exec_exprs.lhs_ordering_expr_ctxs()), &_is_asc_order,
-                                                   &_is_null_first, _offset, _limit, SIZE_OF_CHUNK_FOR_TOPN);
+        if (_limit <= 1024) {
```

Comment:
Add comment for why we use `HeapChunkSorter` when _limit <= 1024, use `ChunksSorterTopn` when _limit > 1024